dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties

```diff
@@ -16,7 +16,7 @@
 
 description=A secure, REST administrative connector that can be used locally or remotely using any Java \
 developer kit. This connector enables remote access from a Java client or directly through an HTTPS call. \
-It is protected by a single administrator role and requires SSL to keep the communication confidential. \
+It is protected by the administrator and the reader role and requires SSL to keep the communication confidential. \
```

Comment:
Change,
"It is protected by the administrator and the reader role and requires SSL to keep the communication confidential. "
to
"The administrator and reader roles protect the connector and it requires an SSL protocol to keep the communication confidential."

Reply:
Ok. Those changes look good. I will update the text.

Reply:
I am going to modify SSL to TLS and remove protocol as TLS implies a protocol and it sounds redundant.

Reply:
Okay. Thanks.

Reply:
@jvanhill Just to clarify. With reference to  'The administrator and reader roles protect the connector '; is it still s single administrator role and an additional or optional reader role?

Reply:
@ManasiGandhi - These are two separate administrative roles. Both are optional. The administrator-role has all permission that the reader-role has.